ui/src/timeMachine/components/Vis.tsx

```diff
@@ -151,6 +152,7 @@ const mstp = (state: AppState): StateProps => {
   const symbolColumns = getSymbolColumnsSelection(state)
 
   const timeZone = state.app.persisted.timeZone
+  const text = get(viewProperties, 'queries[0].text', '')
```

Comment:
why isn't this being parsed out into an object before being put into the reducer?